src/main/java/org/fogbowcloud/saps/engine/core/catalog/Catalog.java

```diff
@@ -0,0 +1,49 @@
+package org.fogbowcloud.saps.engine.core.catalog;
+
+import java.sql.SQLException;
```

Comment:
this import (line #3) is not necessary

Reply:
@ThiagoWhispher please, configure your IDE to auto-detect this.

Reply:
Done